api/nameguard/utils.py

```diff
@@ -7,8 +7,8 @@
 from nameguard.exceptions import InvalidNameHash, InvalidTokenID, InvalidEthereumAddress
 from nameguard.models import Rating, GenericCheckResult, Check
 
-MAX_INSPECTABLE_NAME_LENGTH = 200
-MAX_NUMBER_OF_LABELHASHES_IN_NAME = 5
+MAX_INSPECTABLE_NAME_CHARACTERS = 200  # number of characters to inspect in a name including labels and separators
```

Comment:
```suggestion
MAX_INSPECTED_NAME_CHARACTERS = 200  # maximum number of characters to inspect in a name including labels and separators
```

Suggesting that we move away from using the term "inspectable" and instead use "inspected". This is because "inspectable" seems to suggest it is completely impossible in the abstract, while "inspected" in this context suggests that it may theoretically be possible to inspect something longer, but we just decide not to do it.